src/app/auth/component/login-form/login-form.component.html

```diff
@@ -19,12 +29,24 @@
       type="password"
       placeholder="Password"
     >
-    <mat-error *ngIf="!!formPassword?.errors?.required">
-      Please enter a value
-    </mat-error>
+    <mat-error *ngIf="!!formPassword?.errors?.required">Please enter a value</mat-error>
   </mat-form-field>
 
-  <button mat-raised-button type="submit">
-    LOGIN
-  </button>
+  <div class="buttons">
+    <button
+      mat-raised-button
+      type="submit"
+    >
+      LOG IN
+    </button>
+
+    <button
+      disabled
```

Comment:
I was following the UI Design here: https://github.com/RADAR-base/RADAR-Design/blob/master/Screens/%5BDesktop%5D%20100_Login.png

Is there going to be a signup or has this feature been removed?

Reply:
Technically speaking NO. Because the accounts used in Dashboard are the accounts created in MP and as of now we do not allow account registration. It is created by a user. So for dashboard a user should always have an account that is already created and use those credentials here.

Reply:
Okay sure. I'll remove it. Thanks.